internal/common/resource.go

```diff
@@ -158,6 +158,12 @@ func (a ComputeResourcesFloat) LimitWith(limit ComputeResourcesFloat) ComputeRes
 	return targetComputeResource
 }
 
+func (a ComputeResourcesFloat) LimitTo0() {
```

Comment:
There might be better name for this, maybe spelling Zero is better, or `MakePositive()`?